setup.py

```diff
@@ -122,7 +122,9 @@ def str_from_file(name):
     # Licence: BSD-3-Clause
     "dask[distributed]",
     # Licence: BSD-3-Clause
-    "bokeh!=3.0.*,>=2.4.2"
+    "bokeh!=3.0.*,>=2.4.2",
+    # License: MIT
+    "pydantic"
```

Comment:
Should this be pinned to a particular max version, which would get updated periodically, but only after testing?

Reply:
That's a good idea. I'll add a pin for pydantic and pydantic_core as I was using those during my testing and development